bitnami/kong/templates/kong-prometheus-rolebinding.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if and .Values.metrics.enabled .Values.metrics.serviceMonitor.enabled }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: {{ template "kong.fullname" . }}-prometheus
+  namespace: {{ .Release.Namespace }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: {{ template "kong.fullname" . }}-prometheus
+subjects:
+  {{- if .Values.metrics.serviceMonitor.namespace }}
+  - namespace: {{ .Values.metrics.serviceMonitor.namespace }}
+  {{- else }}
+  - namespace: {{ .Release.Namespace }}
+  {{- end }}        
+  kind: ServiceAccount
+  {{- if .Values.metrics.serviceMonitor.serviceAccount }}
+  name: {{ .Values.metrics.serviceMonitor.serviceAccount }}
+  {{- else }}
+  name: {{ template "kong.serviceAccount" . }}
```

Comment:
Why do we want to associate the Role for Prometheus with Kong's serviceAccount?

Reply:
I just added to be like this condition in `servicemonitor.yaml`, Do you have a better suggestion? I am thinking about removing it

```
  {{- if .Values.metrics.serviceMonitor.namespace }}
  namespace: {{ .Values.metrics.serviceMonitor.namespace }}
  {{- else }}
  namespace: {{ .Release.Namespace }}
  {{- end }}
```

Reply:
I would make the `.Values.metrics.serviceMonitor.serviceAccount` parameter mandatory when `metrics.serviceMonitor.rbac.create` is set to `true`, instead of binding the role to the Kong's ServiceAccount.

Reply:
I have updated the file